### DISCUSSION TITLE
Should we differentiate splitting and aggregation for iteration

### DISCUSSION THREAD

**Discussion by liutiming** [ropensci-books/targets#33] [Ideas]
Hi Will,

In the manual it says 
> The iteration argument of tar_target() controls the splitting and aggregation protocol on a target-by-target basis

Could we say something like "iterations determines how the current target is aggregated, and split the current target for downstream analysis"? When we lump both splitting and aggregating together, I would not be 100% sure what is the iteration parameter is doing to the target that I am writing. 

This understanding comes from using `iteration = "group"` before `iteration = "list"`, where the first iteration splits for the second target, and the second iteration determines how the results will be returned, and how it will be split next.

**Comment by wlandau** [comment]
Sure, I just edited that section in a0bc54b0ebedf419d7a4517f713940dd80dde47f. Hopefully it is clearer now.

**Reply by liutiming** [reply]
Oh yes definitely. It helped improve my understanding of stems and patterns as well. Thanks!
